src/visualize_paths.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+//jscs:disable requireSpacesInForStatement
+//jscs:disable requireSpaceBeforeBinaryOperators
+//jscs:disable disallowMultipleVarDecl
+//jscs:disable validateQuoteMarks
+//jscs:disable requireSpaceBeforeObjectValues
+//jscs:disable validateIndentation
+//jscs:disable requireCurlyBraces
+//jscs:disable requireSpaceAfterKeywords
+//jscs:disable requireSpaceBetweenArguments
+//jscs:disable requireSpacesInConditionalExpression
+//jscs:disable requireSpaceAfterBinaryOperators
+//jscs:disable disallowSpaceAfterPrefixUnaryOperators
+//jscs:disable validateParameterSeparator
+//jscs:disable requireSpaceBeforeBlockStatements
+//jscs:disable disallowSpacesInsideObjectBrackets
+//jscs:disable disallowSpacesInAnonymousFunctionExpression
+//jscs:disable disallowMultipleLineBreaks
+//jscs:disable disallowSpacesInFunctionExpression
+global.visualizePaths = function () {
+  let Visual = require('visual');
+  let colors = [];
+  let COLOR_BLACK = colors.push('#000000') - 1;
+  let COLOR_RED = colors.push('rgba(249,8,8,0.5)') - 1;
+  let COLOR_PATH = colors.push('rgba(255,255,255,0.5)') - 1;
+  _.each(Game.rooms, (room, name) => {
+    let visual = new Visual(name);
+    visual.defineColors(colors);
+    visual.setLineWidth = 0.5;
+    _.each(Game.creeps, creep=> {
+      if (creep.room != room) return;
+      let mem = creep.memory;
+      if (mem._move) {
+        let path = Room.deserializePath(mem._move.path);

```

Comment:
Not sure what this is about. Never looked into `_move` and especially not `_move.path`. I'm only using the `room.memory.routing` paths.

Could be fine, don't know.